core/src/wallet.proto

```diff
@@ -153,6 +156,8 @@ message TransactionInput {
   optional uint32 sequence = 4;
   // Value of connected output, if known
   optional int64 value = 5;
+  // script witness
+  optional ScriptWitness witness = 10;
```

Comment:
Old habit when editing protobuf files, there's no real reason here I'll change it to 6